### PR TITLE
fix(ci): actually run ingest consumer kafka tests take 2

### PR DESCRIPTION
**Pull request by joshuarli**
Brings https://github.com/getsentry/sentry/pull/34907 back. Actual fix is in getsentry: https://github.com/getsentry/getsentry/pull/7562

Also, `make build-platform-assets` is covered by docker builds when we build the wheel: `python setup.py bdist_wheel --build-number 0`. It isn't a SENTRY_LIGHT_BUILD and so does `self.run_command("build_integration_docs")`.

cc @jan-auer for https://github.com/getsentry/sentry/pull/34954#issuecomment-1136768244

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
...
 	@echo ""
 
 test-snuba:
 	@echo "--> Running snuba tests"
-	pytest tests/snuba tests/sentry/eventstream/kafka tests/sentry/snuba tests/sentry/search/events -vv --cov . --cov-report="xml:.artifacts/snuba.coverage.xml" --junit-xml=".artifacts/snuba.junit.xml"
+	pytest tests/snuba \
+		tests/sentry/eventstream/kafka \
+		tests/sentry/snuba \
+		tests/sentry/search/events \
+		tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
```
[the remark this answers is not in the document]
Comment (jan-auer) [reply Makefile:136]: You could cherry-pick this: https://github.com/getsentry/sentry/pull/34954/commits/a2048f63cbdee234dc11405270eb419ed43f83bb

File: Makefile [hunk 2]
```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
...
-	pytest tests/integration tests/sentry
-
 test-python-ci:
-	make build-platform-assets
 	@echo "--> Running CI Python tests"
 	pytest tests/integration tests/sentry \
 		--ignore tests/sentry/eventstream/kafka \
 		--ignore tests/sentry/snuba \
 		--ignore tests/sentry/search/events \
+		--ignore tests/sentry/ingest/ingest_consumer/test_ingest_consumer_kafka.py \
```
Comment (armenzg) [inline Makefile:126]: Commenting for helping my own understanding.

This test is skipped here because it's been xfailing.
> reason: test requires SENTRY_KAFKA_HOSTS environment variable which is not set

File: Makefile [hunk 3]
```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
 	@yarn run test-ci
 	@echo ""
 
-test-python:
-	@echo "--> Running Python tests"
-	# This gets called by getsentry
-	pytest tests/integration tests/sentry
```
Comment (armenzg) [inline Makefile:123]: Why is this target being removed?

File: Makefile [hunk 4]
```diff
@@ -117,24 +117,24 @@ test-js-ci: node-version-check
...
-
 test-python-ci:
-	make build-platform-assets
```
Comment (armenzg) [inline Makefile:126]: Is this a related change to this PR? If not, would it make sense to separate into a different PR?

I'm trying `python setup.py bdist_wheel --build-number 0`, where is that executed? And how does it call `from sentry.utils.integrationdocs import sync_docs; sync_docs()`?

File: Makefile (hunk 3 above)
Comment (joshuarli) [reply Makefile:123]: It isn't used elsewhere after https://github.com/getsentry/getsentry/pull/7562. Also people are most likely not running that many tests at once in local dev.

File: Makefile (hunk 4 above)
Comment (joshuarli) [reply Makefile:126]: No, not related, just a small cleanup. It's so small, I think it's fine to just toss into this PR.

Comment (joshuarli) [reply Makefile:126]:
```
docker/builder.sh
15:python setup.py bdist_wheel --build-number 0
```

It runs this stuff in setup.py:

```
class SentryBuildCommand(BuildCommand):
    def run(self):
        from distutils import log as distutils_log

        distutils_log.set_threshold(distutils_log.WARN)

        if not IS_LIGHT_BUILD:
            self.run_command("build_integration_docs")
            self.run_command("build_assets")
            self.run_command("build_js_sdk_registry")
        BuildCommand.run(self)
```

That's the `run_command("build_integration_docs")`.

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (joshuarli) [reply Makefile:136]: Done.

**Review by jan-auer — APPROVED** [review]

**Review by armenzg — APPROVED** [review]
